roles/openshift_certificate_expiry/test/test_fakeopensslclasses.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+'''
+ Unit tests for the FakeOpenSSL classes
+'''
+
+import os
+import sys
+import unittest
+import pytest
+
+# Disable import-error b/c our libraries aren't loaded in jenkins
+# pylint: disable=import-error,wrong-import-position
+# place class in our python path
+module_path = os.path.join('/'.join(os.path.realpath(__file__).split('/')[:-1]), 'library')
```

Comment:
Won't block on it but when you find the need to use file system separators ``os.path.sep`` is generally a good idea.